AWSLambdaAPI/Models/AWS_ToDo.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace AWSLambdaAPI.Models
+{
+    public class AWS_ToDo
```

Comment:
models are named similar to Controllers, i.e PascalCase

Reply:
Kept it like this because, for future planning to implement Repository pattern for selection/injection of Models (either EF Core generated one, or the One created in ADO.NET or this one is to be used. Base Class/Interface will be added. This API project is unloaded from main solution.